javalin/src/main/java/io/javalin/core/PathParser.kt

```diff
@@ -8,10 +8,10 @@ package io.javalin.core
 
 import io.javalin.http.util.ContextUtil
 
-class PathParser(path: String) {
+class PathParser(path: String, ignoreTrailingSlashes: Boolean) {
 
     val segments: List<PathSegment> = path.split("/")
-            .filter { it.isNotEmpty() }
+            .filter { when { ignoreTrailingSlashes -> it.isNotEmpty() else -> true }}
```

Comment:
This would be better as an if/else

Reply:
I miss your review.
I will update in a few days

Reply:
done 👍